src/main/kotlin/com/bridgecrew/ui/rightPanel/dictionaryDetails/SecretsDictionaryPanel.kt

```diff
@@ -0,0 +1,25 @@
+package com.bridgecrew.ui.rightPanel.dictionaryDetails
+
+import com.bridgecrew.results.SecretsCheckovResult
+
+class SecretsDictionaryPanel(result: SecretsCheckovResult): DictionaryExtraInfoPanel() {
+    override var fieldsMap: MutableMap<String, Any?> = mutableMapOf(
+            "Severity" to result.severity,
+            "Description" to result.description,
+            "Code" to extractCode(result),
+            "Category" to result.category,
+            "Check Type" to result.checkType.name,
+    )
+    init {
+        addCustomPolicyGuidelinesIfNeeded(result)
+        createDictionaryLayout()
+    }
+
+    private fun extractCode(result: SecretsCheckovResult): Any {
+        return try {
```

Comment:
is the code here always small? because we might need a text area of some sort if not (see the diff panel for example)

Reply:
It's a secret so should be one line :)